### PR TITLE
Added script to hash mutant diffs (fixes #242)

### LINKED ISSUE #242: Assign (stable) unique identifiers to mutants

### PR DESCRIPTION
**Pull request by ChrisTimperley**

### CODE REVIEW

File: experiments/hash_mutants.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+if __name__ == '__main__':
+    args = get_args()
+    with open(args.input, 'r') as f:
+        db = YAML().load(f)
+
+    for entry in db['entries']:
+        entry['uid'] = mutation_to_uid(entry['diff'])
+        entry['diff'] = PSS(entry['diff'])
```
Comment (afsafzal) [inline experiments/hash_mutants.py]: Is this necessary?

Comment (ChrisTimperley) [reply experiments/hash_mutants.py]: I think so, yeah. If we don't use `PSS`, we'll end up with difficult-to-read `diff` properties.

Comment (afsafzal) [reply experiments/hash_mutants.py]: yeah yeah. but the `diff` already is `PSS` (if we assume only our scripts are making these files). Anyways, doesn't matter that much.

Comment (ChrisTimperley) [reply experiments/hash_mutants.py]: I just dropped `PSS`, and you're right: the original formatting is preserved.

File: experiments/generate_mutants.py [hunk 2]
```diff
@@ -151,7 +153,12 @@ def main():
                 diff = str(client_boggart.mutations_to_diff(snapshot, [mutation]))
                 acceptable.append(diff)
 
-        yml = [{'diff': PreservedScalarString(d)} for d in acceptable]
+        yml = []
+        for diff in acceptable:
+            entry = {'diff': PreservedScalarString(diff),
+                     'uid': mutation_to_uid(diff)}
```
Comment (afsafzal) [inline experiments/generate_mutants.py]: @ChrisTimperley  just to make sure, isn't `mutation_to_uid(diff)` going to be different than `mutation_to_uid(PSS(diff))`? if so our scripts are going to be inconsistent.

**Pull request merged**